### PR TITLE
Issue #3213188 by agami4: Improve favicon on the admin pages, improve logos on the gin toolbar

### PR DESCRIPTION
**Pull request by agami4**
## Problem
The admin pages do not have the correct favicon.
Should change logos on the top/bottom of the gin toolbar.

## Solution
Improve favicon on the gin theme settings.
Improve logos on the top/bottom of the gin toolbar.

## Issue tracker
https://getopensocial.atlassian.net/browse/DS-7597
https://www.drupal.org/project/social/issues/3213188

## How to test
- [x] Logged like SM and go to the admin page.
- [x] Check the favicon and logo on the top of the gin toolbar.

## Screenshots
before:
image (3)

after:
Structure | ECI Demo 2021-05-10 17-54-54

## Release notes
The admin pages have an Open Social favicon.
The logos were changes on the top/bottom of the gin toolbar

### CODE REVIEW

**Review by navneet0693 — APPROVED** [review]

**Comment by navneet0693** [comment]
🍒 - picked at 46ccfd5b8f23c84429ed5f0d1a252e33d08c059d

File: modules/social_features/social_core/social_core.install [hunk 1]
```diff
@@ -1316,7 +1316,24 @@ function social_core_update_8908() {
...
+    'social_advanced_queue',
   ];
   \Drupal::service('module_installer')->install($modules);
 }
+
+/**
+ * Add a OS favicon on the admin pages.
+ */
+function social_core_update_10102() {
+  $config_file = drupal_get_path('module', 'social_core') . '/config/install/gin.settings.yml';
```
Comment (ronaldtebrake) [inline modules/social_features/social_core/social_core.install:1328]: Hi @navneet0693 @agami4 this is not a good idea, using static files is necessary here.

File: modules/social_features/social_core/config/install/gin.settings.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+favicon:
```
Comment (ronaldtebrake) [inline modules/social_features/social_core/config/install/gin.settings.yml:1]: Hi @agami4 @navneet0693 
gin.settings.yml is something that is provided by the gin theme already

so we will be getting a
```
Drupal\Core\Config\PreExistingConfigException: Configuration objects     [error]
(gin.settings) provided by gin already exist in active configuration
in
```

**Comment by ronaldtebrake** [comment]
For now we can fix it in https://github.com/goalgorilla/open_social/pull/2330 as we also noticed these changes dont work with the new gin_toolbar version 🙏

**Pull request merged**